static/app/views/settings/projectMetrics/metricsExtractionRuleForm.tsx

```diff
@@ -306,12 +308,18 @@ export function MetricsExtractionRuleForm({
     [onSubmit]
   );
 
+  const isCustomSpanAttribute = useCallback((value: string) => {
```

Comment:
This will return false for custom span attributes that are already present in the dropdown so i would propose renaming it to:
```suggestion
  const isNewCustomSpanAttribute = useCallback((value: string) => {
```